openquakeplatform/openquakeplatform/vulnerability/views.py

```diff
@@ -117,7 +117,7 @@ def list_entries(request, **kwargs):
     if llrs:
         geninfo = geninfo.filter(llrs=llrs)
     if author:
-        geninfo = geninfo.filter(authors__icontains=author)
+        geninfo = geninfo.filter(authors__unaccent=author)
```

Comment:
This is the ONLY place where the unaccent feature is used? If yes, instead of creating a monster perhaps it would have been possible to make a special case for this, use a raw query with something like

`select ... from table where unaccent(author)=unaccent(%s)`